src/runtime/config/configuration-clh-snp.toml.in

```diff
@@ -43,7 +40,8 @@ rootfs_type=@DEFROOTFSTYPE@
 # Default false
 confidential_guest = true
 
-# enable SEV SNP VMs. This is not currently used by CLH
+# enable SEV SNP VMs.
+# This is not currently used by CLH
```

Comment:
Is the second comment line out of date for our branch?

Reply:
Thank you, revisited the comment